.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,90 @@
+name: Bug report
+description: Bugs in Falcon
+title: "Falcon Issue: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Before** reporting an issue, make sure to read the [documentation](https://picovoice.ai/docs/falcon/) and search [existing issues](https://github.com/Picovoice/falcon/issues).
+  - type: checkboxes
+    id: check
+    attributes:
+      label: Have you checked the docs and existing issues?
+      description: Make sure you have checked all of the below before submitting an issue
+      options:
+        - label: I have read all of the relevant Picovoice Falcon docs
+          required: true
+        - label: I have searched the existing issues for Falcon
+          required: true
+  - type: dropdown
+    id: sdk
+    attributes:
+      label: SDK
+      options:
+        - Python
+    validations:
+      required: true
+  - type: input
+    id: package
+    attributes:
+      label: "Falcon package version"
+      placeholder: "1.0.0"
+    validations:
+      required: true
+  - type: input
+    id: framework
+    attributes:
+      label: "Framework version"
+      placeholder: "Python 3.7, .NET Core 3.1, etc."
+    validations:
+      required: true
+  - type: dropdown
+    id: platform
+    attributes:
+      label: Platform
+      options:
+        - Android
```

Comment:
we dont have support for `android`, `ios`, and `web` and `ARM Cortex-M`, can you remove them?

Actually for Eagle and Koala also remove `ARM Cortex-M`. We only have it for `Porcupine`, `Rhino`, and `Picovoice`, so you can remove it for all other animals